Wikipedia/Code/EditSaveViewController.swift

```diff
@@ -489,9 +489,9 @@ class EditSaveViewController: WMFScrollViewController, Themeable, UITextFieldDel
                   let currentTitle = pageURL?.wmf_title else {
                 return
             }
-            
+            // swiftlint:disable unused_capture_list
```

Comment:
Hmm I'm not seeing this warning - can you try updating swiftlint? The latest I'm able to update to is `0.53.0`.

Reply:
> Hmm I'm not seeing this warning - can you try updating swiftlint? The latest I'm able to update to is `0.53.0`.

I just updated, and they went away! I'll close this PR as the other ones seem to have been fixed as well